packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# 49.7.0 - August 25, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-lightning
+
+- Update to the latest version of the Aura and LWC language servers [PR #2380](https://github.com/forcedotcom/salesforcedx-vscode/pull/2380)
+
+#### salesforcedx-vscode-lwc
+
+- Update to the latest version of the LWC language server [PR #2380](https://github.com/forcedotcom/salesforcedx-vscode/pull/2380)
+
+#### docs
+
+- Fix example under [Advanced Setup](https://developer.salesforce.com/tools/vscode/en/getting-started/java-setup/#advanced-setup) on Java Setup page [PR #2426](https://github.com/forcedotcom/salesforcedx-vscode/pull/2426) - Contribution by ([@nabondance](https://github.com/nabondance))
+
 # 49.5.0 - August 13, 2020
```

Comment:
Did 49.6.0 get left out?

Reply:
Yes, I think we forgot to port that change back to `develop`